configure

```diff
@@ -578,6 +578,7 @@ opt ccache 0 "invoke gcc/clang via ccache to reuse object files between builds"
 opt local-rust 0 "use an installed rustc rather than downloading a snapshot"
 opt llvm-static-stdcpp 0 "statically link to libstdc++ for LLVM"
 opt rpath 0 "build rpaths into rustc itself"
+opt landing-pads 1 "enable landing pads during bootstrap"
```

Comment:
Perhaps this could be called `stage0-landing-pads` to ensure it's not mistaken for disabling landing pads entirely?